packages/odyssey-react/src/components/Banner/Banner.module.scss

```diff
@@ -62,28 +62,28 @@
 }
 
 .isDismissable {
-  padding-block: $spacing-s;
-  padding-inline-start: $spacing-m;
-  padding-inline-end: $spacing-l;
+  padding-block: var(--SpaceRemS);
+  padding-inline-start: var(--SpaceRemM);
+  padding-inline-end: var(--SpaceRemL);
 }
 
 .infoVariant {
-  background-color: cv("blue", "300");
+  background-color: var(--ColorPrimaryLight);
 }
 
 .successVariant {
-  background-color: cv("green", "300");
+  background-color: var(--ColorSuccessLight);
 }
 
 .cautionVariant {
-  background-color: cv("yellow", "300");
+  background-color: var(--ColorCautionLight);
 }
 
 .dangerVariant {
-  background-color: cv("red", "300");
+  background-color: var(--ColorDangerLight);
 }
 
-@media screen and (max-width: ($max-line-length * 1.5)) {
+@media screen and (max-width: (calc(var(--LayoutMaxlinelength) * 1.5))) {
```

Comment:
ya good catch, because we're using a css custom property now we need to have `calc` available at run time to hand this transformation were before it could be compiled at build time by sass.

Curious: Did you get a helpful error to catch this initially?

Reply:
No error, but I caught it on the first run with a `\$` grep - been my standard for making sure no variables go untouched.